atlas/src/test/java/us/ihmc/atlas/AtlasLiftOffAndTouchDownTest.java

```diff
@@ -36,6 +41,11 @@ public double nominalHeightAboveAnkle()
          }
       };
       SCS2AvatarTestingSimulation testHelper = AvatarLiftOffAndTouchDownTest.setupTest(robotModel, Math.toRadians(90.0));
+      // We need to turn off the CoP threshold fraction here. If it's left on, contact isn't triggered until the CoP is far enough within the foot. This means
+      // that the detection of contact is significantly delayed. It then misses the cotnact timing, and also ahs a tendency to "slap" the foot onto the ground,
+      // rather than smoothly loading it.
+      String namespace = HighLevelHumanoidControllerFactory.class.getSimpleName();
+      ((YoDouble) testHelper.findVariable(namespace, "CoPThresholdFraction")).set(Double.NaN);
```

Comment:
Without this, the contact isn't detected until its too late because the CoP is on the edge.